src/components/AddressSearch/index.js

```diff
@@ -238,7 +240,10 @@ const AddressSearch = (props) => {
                         defaultValue: props.defaultValue,
                         inputID: props.inputID,
                         shouldSaveDraft: props.shouldSaveDraft,
-                        onBlur: props.onBlur,
+                        onBlur: (event) => {
+                            resetDisplayListViewBorderOnBlur(event, containerRef, setDisplayListViewBorder);
+                            props.onBlur(event);
```

Comment:
```suggestion
                            props.onBlur(event);
```

Do we need to pass `event` to `onBlur` here?